domains/skythegmer.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "skyy-codes",
+        "email": "skyxgamiing@gmail.com"
+    },
+    "record": {
+        "CNAME": "google.com"
```

Comment:
please replace google.com with your site

Reply:
I'll do that in a sec